Mathlib/SetTheory/Cardinal/SchroederBernstein.lean

```diff
@@ -68,7 +71,22 @@ theorem schroeder_bernstein {f : α → β} {g : β → α} (hf : Function.Injec
       obtain ⟨y', hy', rfl⟩ : y ∈ g '' (f '' s)ᶜ := by rwa [hns]
       rw [g'g _] at hxy
       exact hy' ⟨x, hx, hxy⟩
-  exact ⟨h, ‹Injective h›, ‹Surjective h›⟩
+  refine ⟨h, ⟨‹Injective h›, ‹Surjective h›⟩, fun a ↦ ?_⟩
+  simp only [h, Set.piecewise, g']
+  split
+  · exact hp₁ a
+  · have : a = g (invFun g a) := by
+      have : a ∈ g '' (f '' s)ᶜ := by grind
+      obtain ⟨x, _, hx⟩ := mem_image _ _ _ |>.mp this
+      exact Function.invFun_eq ⟨x, hx⟩ |>.symm
```

Comment:
```suggestion
      exact Function.invFun_eq ⟨x, hx⟩
```
... then here you do not need to reverse the equality!

Reply:
Thanks!